crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -781,9 +813,90 @@ impl LogsController {
             metadata_store_client,
             metadata_writer,
             async_operations: JoinSet::default(),
+            tail_updates_tx,
+            in_flight_refresh_tail: None,
         })
     }
 
+    pub fn watch_logs_tail_updates(&self) -> watch::Receiver<Arc<LogsTailUpdates>> {
+        self.tail_updates_tx.subscribe()
+    }
+
+    pub fn schedule_refresh_tails(&mut self) -> Result<(), ShutdownError> {
```

Comment:
Maybe this is a responsibility we can encapsulate into Bifrost admin in the future, to provide a more convenient way to fetch the `TailState`s across a set of logs? There may be other code components interested in watching for reconfiguration besides the cluster controller.